### ISSUE TITLE
Null safety issue in accessibility.dart

### ISSUE THREAD

**Issue by kendalsickels-wf** [xvrh/puppeteer-dart#194]
We've been a bit behind in our dependency so I'm only recently getting around to actually consuming null safe versions of puppeteer-dart. Upon doing so I'm hitting the following stack in v2.9.0 (can't upgrade to newer due to dep issues on our end):

```
  Null check operator used on a null value
  package:puppeteer/protocol/accessibility.dart 592:65  new AXPropertyName.fromJson
  package:puppeteer/protocol/accessibility.dart 435:28  new AXProperty.fromJson
  package:puppeteer/protocol/accessibility.dart 665:38  new AXNodeData.fromJson.<fn>
  dart:_internal                                        ListIterable.toList
  package:puppeteer/protocol/accessibility.dart 666:16  new AXNodeData.fromJson
  package:puppeteer/protocol/accessibility.dart 72:32   AccessibilityApi.getFullAXTree.<fn>
  dart:_internal                                        ListIterable.toList
  package:puppeteer/protocol/accessibility.dart 73:10   AccessibilityApi.getFullAXTree
```

Doing a little debugging it appears there are values we're getting back from the json that are not in the enum (formerly a const list). In my case I'm getting these:
```
'notVisible'
'labelFor'
'presentationalRole'
```

I could put up a change that adds to the enum but I fear that it could need more additions in the future. Curious if we should be passing the fromJson value directly to `AXPropertyName` vs using an enum or const list?

```
factory AXPropertyName.fromJson(String value) => AXPropertyName(value);
```

Additionally, would you be opposed to back patching whatever we decide to do so that I can use it 😬 ?

fyi @xvrh

**Comment by kendalsickels-wf** [comment]
Quick addition based off [master](https://github.com/xvrh/puppeteer-dart/compare/master...kendalsickels-wf:puppeteer-dart:null-safety-fix?expand=1) that should allow us to keep working. 

And a similar [change](https://github.com/xvrh/puppeteer-dart/compare/v2.9.0...kendalsickels-wf:v2.9-null-safety-fix?expand=1) based off of v2.9.0.

Because we've been using an older version of Dart I haven't played with Enhanced Enum Classes to know if they can work with somewhat dynamic content but if not maybe going back to a standard class would be best 🤷

**Comment by xvrh** [comment]
@kendalsickels-wf yes, I've also seen missing enums in the protocol before.
Generally we apply a workaround/hack/fix in the code generator here: https://github.com/xvrh/puppeteer-dart/blob/master/tool/generate_protocol.dart#L788. I think you can also add your properties there...
